commit_log/.gitignore

```diff
@@ -8,3 +8,6 @@ target/
 
 ## Cargo lock
 Cargo.lock
+
+# IDE
+**/.idea/
```

Comment:
I don't mind it much, but you could move this to your global git ignore since it is personal preference.